diff -ur original/cocos2d-2.1rc0-x-2.1.3/cocos2dx/support/zip_support/ioapi.h cocos2d-2.1rc0-x-2.1.3/cocos2dx/support/zip_support/ioapi.h
--- cocos2d-2.1rc0-x-2.1.3/cocos2dx/support/zip_support/ioapi.h	2013-04-28 10:06:40.000000000 +0200
+++ cocos2d-2.1rc0-x-2.1.3/cocos2dx/support/zip_support/ioapi.h	2013-05-23 21:44:43.339985692 +0200
@@ -18,6 +18,8 @@
 
 */
 
+#define OF(x) x
+
 #ifndef _ZLIBIOAPI64_H
 #define _ZLIBIOAPI64_H
 
diff -ur original/cocos2d-2.1rc0-x-2.1.3/cocos2dx/support/zip_support/unzip.h cocos2d-2.1rc0-x-2.1.3/cocos2dx/support/zip_support/unzip.h
--- cocos2d-2.1rc0-x-2.1.3/cocos2dx/support/zip_support/unzip.h	2013-04-28 10:06:40.000000000 +0200
+++ cocos2d-2.1rc0-x-2.1.3/cocos2dx/support/zip_support/unzip.h	2013-05-23 21:44:43.339985692 +0200
@@ -40,6 +40,8 @@
 
 */
 
+#define OF(x) x
+
 #ifndef _unz64_H
 #define _unz64_H

